runtime/Cpp/runtime/src/misc/Interval.h

```diff
@@ -26,59 +26,56 @@ namespace misc {
     ssize_t b;
 
     Interval();
-    explicit Interval(size_t a_, size_t b_); // For unsigned -> signed mappings.
```

Comment:
I missed that one in my first review. Why did you remove the `explicit` keyword? I certainly don't want automatic conversion between `size_t` and `ssize_t`! Compilers behave differently here. Please put that back.

Reply:
Seems we haven't finished this one. Are you sure we don't need the `explicit` keyword?

Reply:
It doesn't do what you think, re: signed vs. unsigned comparisons, however, adding it back in because it doesn't hurt.